src/manager.js

```diff
@@ -0,0 +1,116 @@
+const Cohort = require('./cohort')
+
+class Manager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  addCohort(cohortName) {
+    const cohort = new Cohort(cohortName)
+    this.cohorts.push(cohort)
+  }
+
+  removeCohortByName(cohortName) {
+    const cohortsByName = []
+    for (let j = 0; j < this.cohorts.length; j++) {
+      cohortsByName.push(this.cohorts[j].cohortName)
+    }
+    if (cohortsByName.includes(cohortName)) {
+      for (let j = 0; j < this.cohorts.length; j++) {
+        if (this.cohorts[j].cohortName === cohortName) {
+          this.cohorts.splice(j, 1)
+        }
+      }
+      return this.cohorts
+    }
+    if (!cohortsByName.includes(cohortName)) {
+      return `No such cohort`
+    }
+  }
+
+  createUniqueId(gitHub, cohortName) {
+    return `${gitHub}_${cohortName.slice(-5)}`
+  }
+
+  addStudentCohortUniqueId(firstName, lastName, gitHub, cohortName) {
+    const uniqueId = this.createUniqueId(gitHub, cohortName)
+
+    let addedCohorts = null
+    this.cohorts.forEach((cohort, i) => {
+      if (cohortName === cohort.cohortName) {
+        cohort.addStudentToCohort(firstName, lastName, gitHub, uniqueId)
+        addedCohorts = cohort
+      }
+    })
+    if (addedCohorts === null) {
+      return 'Not found'
+    }
+
+    return addedCohorts
+  }
```

Comment:
line 35: in my opinion making the function name addStudentToCohort or addStudentUniqueId**To**Cohort would be a better name - reading that function initially made me think it was going to add another cohort - especially with the addedCohorts variable in there

line 39: whats the i doing here? i can't see it being used anywhere else in the function